src/app/(protected)/mumble/[mumbleId]/page.tsx

```diff
@@ -0,0 +1,22 @@
+import { Suspense } from "react";
+import { PostDetailPage } from "@/components/post-detail-page";
+
+type MumblePageProps = {
+  params: Promise<{
+    mumbleId: string;
+  }>;
+};
+
+const MumblePage = async ({ params }: MumblePageProps) => {
+  const { mumbleId } = await params;
+
+  return (
+    <main>
```

Comment:
Nice `<main>` 👍

OK for now, maybe we should check later where we should add `<main>` and where `<section>`. 

A11y tests will tell us what we have to do. 😊